Makefile

```diff
@@ -84,7 +84,7 @@ rhel-release:
 
 .PHONY: rhel-nightly
 rhel-nightly:
-	./hack/go-build.sh ${BUILDER} rhel nightly
+	./hack/go-build.sh ${BUILDER} rhel nightly $(build_user)
```

Comment:
I've got a doubt, is it correct to use ```$(build_user)``` instead of ```${build_user}```

Reply:
$() expect it as parameter from make , e.g.

make test build_user=user1

While ${} is a env.